src/components/common/PageWithModuleTimeline/ModuleTimeline.tsx

```diff
@@ -55,8 +70,23 @@ const ModuleTimeline = ({ cohortPhase, modules }: ModuleTimelineProps) => {
           {(modules ?? []).map((module, index) => (
             <Step key={module.id} active={module.isActive}>
               <StepLabel
-                icon={<AltStepIcon isActive={module.isActive} index={index} />}
-                sx={{ fontWeight: 600, '.MuiStepLabel-label.Mui-disabled': { fontWeight: 600 } }}
+                icon={
+                  <AltStepIcon
+                    index={index}
+                    bgColor={
+                      soonestEndingModuleId === module.id
+                        ? theme.palette.TwClrBgWarning
+                        : module.isActive
+                          ? theme.palette.TwClrBaseGreen500
+                          : theme.palette.TwClrBaseGray300
+                    }
+                  />
+                }
+                sx={
+                  soonestEndingModuleId === module.id
+                    ? warningLabelStyles
+                    : { fontWeight: 600, '.MuiStepLabel-label.Mui-disabled': { fontWeight: 600 } }
```

Comment:
Would it makes sense to inject `fontColor` here directly? instead of making a style class?

Reply:
Ok to submit either way! Mostly asking because I am not too good with CSS

Reply:
Without the classes the color is override by mui active / disabled colors